apps/emqx/src/emqx_trie.erl

```diff
@@ -249,6 +276,17 @@ insert_key(Key, Trie) ->
         end,
     ok = mnesia:write(Trie, T, write).
 
+insert_key_local(Key, Dest, Trie) ->
+    T =
+        case ets:lookup(Trie, Key) of
+            [#?TRIE{count = Ds} = T1] ->
+                T1#?TRIE{count = Ds#{Dest => []}};
```

Comment:
How possible concurrency issues can be resolved?
M.b. we could use an ordered ets with `Dest` being a part of the key?

Reply:
Do we rely on the fact that there is a single table subscriber doing updates?

Reply:
Yes. This is why it's not really critical right now, so that we could look at the impact of removing transactions from the wildcard-subscription codepath without revamping the indexing too much.